debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
With the snapshot flag I get:

```
mu (0.1-2~1.gbp609e30) UNRELEASED; urgency=medium

  ** SNAPSHOT build @609e30ea818e5e45cede2b7394b6bd8f64b367ad **

  * Add setuptools to package dependencies.

 -- carlosperate <carlosperate@users.noreply.github.com>  Thu, 21 Apr 2016 07:25                          :55 +0000

mu (0.1-1) unstable; urgency=low
...
```

Without the snapshot flag the message is very similar (without any mentions to it being a snapshot), and the version is a simple `mu (0.1-2) UNRELEASED; urgency=medium`.